plugins/jira/reference/rosa.md

```
@@ -0,0 +1,388 @@
+| Service Lifecycle | scrum | 11897 | `16c10b5d-b9d1-4b9a-949d-39888ef9455a` |
+| Thor | scrum | 11898 | `51fd486e-45a1-48de-955e-57899973d2b3` |
+
+**JQL example:**
+```jql
+project = ROSAENG AND Team = "b1d72bb9-7e1c-4fc4-96f0-44eb3aebea7e" ORDER BY Rank ASC
+```
+
+The UI renders the friendly name (e.g., "[ROSA] Coffee") but JQL requires the UUID.
+
+**Board URL pattern:**
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)